assets/js/translator.js

```diff
@@ -223,6 +223,8 @@ if(modeEnabled('translation')) {
                 $('#fileInput').show();
                 $('div#fileName').hide();
                 $('div#docTranslation').fadeIn('fast');
+                $('#detect').prop('disabled', true);
+                $('#srcLangSelect').children('option[value=' + 'detect' + ']').attr('disabled', true);
```

Comment:
Um @sushain97 , are you suggesting something like following? (using a jquery inside another selector)?  The `option` value does not have an ID. Thus I'll need to select `detect` from the `select` dropdown and disable the option. Referred [this](https://stackoverflow.com/questions/10525651/jquery-selector-element-inside-other)

`$('#detect', $('#srcLangSelect').children('option[value=detect]')).attr('disabled', true);`